applications/OptimizationApplication/tests/algorithm_tests/test_convergence.py

```diff
@@ -0,0 +1,29 @@
+from abc import ABC, abstractmethod
+
+import KratosMultiphysics as Kratos
+import KratosMultiphysics.KratosUnittest as kratos_unittest
+from KratosMultiphysics.OptimizationApplication.utilities.opt_convergence import *
+from KratosMultiphysics.OptimizationApplication.utilities.optimization_problem import OptimizationProblem
+
+class TestConvergence(kratos_unittest.TestCase, ABC):
```

Comment:
```suggestion
class TestConvergence(kratos_unittest.TestCase):
```

No need to derrive from `ABC`